front/pages/w/[wId]/builder/data-sources/index.tsx

```diff
@@ -344,96 +347,48 @@ export default function DataSourcesView({
       topNavigationCurrent="settings"
       subNavigation={subNavigationAdmin({ owner, current: "data_sources" })}
     >
-      <PageHeader
-        title="Data Sources"
-        icon={CloudArrowDownIcon}
-        description="Control the data Dust has access to and how it's used."
-      />
-
-      <div>
-        <SectionHeader
-          title="Managed Data Sources"
-          description="Give Dust real-time data updates by linking your company's online knowledge bases."
-        />
-
-        <div>
-          <ul role="list" className="mt-4 divide-y divide-structure-200">
+      <Page.Vertical gap="lg" align="stretch">
+        <div className="flex flex-col gap-4 pb-4">
+          <PageHeader
+            title="Data Sources"
+            icon={CloudArrowDownIcon}
+            description="Control the data Dust has access to and how it's used."
+          />
+
+          <SectionHeader
+            title="Managed Data Sources"
+            description="Give Dust real-time data updates by linking your company's online knowledge bases."
+          />
+
+          <ContextItem.List className="mt-2">
```

Comment:
nit: mt-2 not needed to match figma, which indicates gap-3 (in theory it should even be -mt-1 to compensate for the gap-4--but cf below similar comment, consistency of gap-4 is also ok)